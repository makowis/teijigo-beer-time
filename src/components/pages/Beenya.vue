<template>
  <tbt-default>
    <div class="beenya">
      <tbt-h1>びーにゃの部屋</tbt-h1>
      <tbt-beenya-img :class="$style.icon" />
      <table :class="$style.beenyaTable">
        <tbody>
          <tr>
            <th>名前</th>
            <td>びーにゃ</td>
          </tr>
          <tr>
            <th>役割</th>
            <td>アイドル</td>
          </tr>
          <tr>
            <th>自己紹介</th>
            <td>
              にゃにゃにゃにゃにゃにゃにゃー！(定時後ビールタイムだー！！)
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </tbt-default>
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';
import TbtDefault from '@/components/templates/TbtDefault.vue';
import TbtH1 from '@/components/atoms/TbtH1.vue';
import TbtBeenyaImg from '@/components/atoms/TbtBeenyaImg.vue';

@Component({
  components: {
    TbtDefault,
    TbtH1,
    TbtBeenyaImg,
  },
})
export default class Beenya extends Vue {}
</script>

<style module>
.icon {
  max-width: 90%;
  margin-bottom: 20px;
}

.beenyaTable {
  max-width: 90%;
  margin: auto;
  border: solid 1px black;
  border-collapse: separate;
}

.beenyaTable td,
.beenyaTable th {
  border: solid 1px black;
  padding: 5px 10px;
}
</style>
