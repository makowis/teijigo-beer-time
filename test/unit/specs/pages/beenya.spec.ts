import { mount, RouterLinkStub } from '@vue/test-utils'
import Beenya from '@/pages/beenya.vue';

describe('beenya.vue', () => {
  const vm = mount(Beenya, {
    stubs: {
      NuxtLink: RouterLinkStub
    }
  }).vm;  

  it('should match the snapshot', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
