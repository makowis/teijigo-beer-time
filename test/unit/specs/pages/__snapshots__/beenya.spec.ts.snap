// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`beenya.vue should match the snapshot 1`] = `
<div
  class="beenya"
>
  <h1
    class="h1"
  >
    びーにゃの部屋
  </h1>
   
  <img
    alt="びーにゃのイラスト"
    class="icon"
    src="@/assets/beenya.png"
  />
   
  <table
    class="beenyaTable"
  >
    <tbody>
      <tr>
        <th>
          名前
        </th>
         
        <td>
          びーにゃ
        </td>
      </tr>
       
      <tr>
        <th>
          役割
        </th>
         
        <td>
          アイドル
        </td>
      </tr>
       
      <tr>
        <th>
          自己紹介
        </th>
         
        <td>
          にゃにゃにゃにゃにゃにゃにゃー！(定時後ビールタイムだー！！)
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
